README.md

```diff
@@ -22,8 +22,14 @@ fn main() {
     .flag(Some("-d"), Some("--debug"), Some("Activate debug mode"))
     .flag(Some("-v"), Some("--verbose"), Some("Verbose mode"));
     .option(Some("-o"), Some("--output"), "output", None, "Output file");
+
+  let _string = page.to_string();
 }
 ```
+Preview by running:
```

Comment:
I _hate_ that you don't put empty lines around code blocks. Just so you know. :P

Reply:
Thanks for sharing; your comment has been noted :stuck_out_tongue:

Reply:
As a side note: I'll happily switch to whichever style is conventional the moment `rustfmt` is able to do so automatically. Can't be bothered to otherwise :P

Reply:
brb, doing `cargo new mdfmt`